Orange/tests/test_variable.py

```diff
@@ -280,81 +268,74 @@ def test_val(self):
         a = StringVariable("a")
         self.assertEqual(a.to_val(None), "")
         self.assertEqual(a.str_val(""), "?")
-        self.assertEqual(a.str_val(Value(a, "")), "?")
-        self.assertEqual(a.repr_val(Value(a, "foo")), '"foo"')
 
 
 @variabletest(TimeVariable)
 class TestTimeVariable(VariableTest):
+    TODAY_TIMESTAMP = pytz.utc.localize(datetime.combine(datetime.now(), datetime.min.time())).timestamp()
+    TODAY_STRING = str(date.today())
+
     TESTS = [
         # in str, UTC timestamp, out str (in UTC)
-        ('2015-10-12 14:13:11.01+0200', 1444651991.01, '2015-10-12 14:13:11.010000+0200'),
-        ('2015-10-12T14:13:11.81+0200', 1444651991.81, '2015-10-12 14:13:11.810000+0200'),
-        ('2015-10-12 14:13:11.81', 1444659191.81, '2015-10-12 14:13:11.810000'),
-        ('2015-10-12T14:13:11.81', 1444659191.81, '2015-10-12 14:13:11.810000'),
```

Comment:
Why were these two thrown out? The microsecond fraction doesn't parse if no timezeone?